node/node.go

```diff
@@ -172,6 +187,63 @@ func (n *Node) Start() error {
 	return nil
 }
 
+// startRPC initializes and starts the IPC and HTTP RPC endpoints.
+func (n *Node) startRPC(services map[reflect.Type]Service) error {
+	// Gather and register all the APIs exposed by the services
+	ipcHandler := rpc.NewServer()
+	httpHandler := rpc.NewServer()
```

Comment:
Ah, indeed that was a leftover from when I still wanted to do the HTTP in this PR. Will remove the http version from this one.